### DISCUSSION TITLE
Proposal to improve identification in normal and elevate state

### DISCUSSION THREAD

**Discussion by Xanardi** [microsoft/terminal#18202] [Ideas]
I would like to be able to set separate "Appearance settings" for normal and elevate in one profile.

During Elevate, an icon is displayed in the corner of the window, 
but I think the visibility of the icon is not good enough.

I would like to be able to distinguish it more clearly.

Also, I would like to have a function to hide the specified profile from the list when Elevate is selected.
We have created a profile that we do not want to be executed by Elevate by mistake.

**Comment by zadjii-msft** [comment]
Thanks for filing this! We're kinda collecting up all these thoughts in #11111. (make sure to go give it, and linked threads, a 👍) There's a lot of thorny edge cases with this, which has kept it pretty far down the backlog. The current settings model doesn't really have an _elegant_ way to have a profile act differently based on elevation status. 

If you have anything else to add - that'd be the thread to do it in. Thanks!
